### ISSUE TITLE
Pico question (not self-hosted): certificate accepted by `pico.sh` but not by `tuns.sh` (prompting for password)

### ISSUE THREAD

**Issue by diraneyya** [antoniomika/sish#331]
Hello,

I did not deploy this on my own server. I am just using tuns.sh

I have created an account using `ssh pico.sh` then copies all of the public SSH keys from the devices I want to be connecting to with SSH tunnels.

Now, after confirming that I am indeed recognised as the correct user using `ssh pico.sh`, and from the same machine I am writing the following command:
```
ssh -R ansible:22:localhost:22 tuns.sh
root@tuns.sh's password: _
```

... as seen above, I am being prompted for a password. I tried the verbose output and it indeed seems like the same key (which `pico.sh` accepts and associated with my username) is not being accepted by `tuns.sh`:
```
debug2: pubkey_prepare: done
debug1: Offering public key: /Users/orwa/.ssh/id_rsa RSA SHA256:VCg0KIg0upWm/UOmWqtZw0M32u73v5q+R/eQH87YK4U
debug3: send packet: type 50
debug2: we sent a publickey packet, wait for reply
debug3: receive packet: type 51
debug1: Authentications that can continue: password,publickey
debug1: Trying private key: /Users/orwa/.ssh/id_ecdsa
debug3: no such identity: /Users/orwa/.ssh/id_ecdsa: No such file or directory
debug1: Trying private key: /Users/orwa/.ssh/id_ecdsa_sk
debug3: no such identity: /Users/orwa/.ssh/id_ecdsa_sk: No such file or directory
debug1: Trying private key: /Users/orwa/.ssh/id_ed25519
debug3: no such identity: /Users/orwa/.ssh/id_ed25519: No such file or directory
debug1: Trying private key: /Users/orwa/.ssh/id_ed25519_sk
debug3: no such identity: /Users/orwa/.ssh/id_ed25519_sk: No such file or directory
debug1: Trying private key: /Users/orwa/.ssh/id_xmss
debug3: no such identity: /Users/orwa/.ssh/id_xmss: No such file or directory
debug1: Trying private key: /Users/orwa/.ssh/id_dsa
debug3: no such identity: /Users/orwa/.ssh/id_dsa: No such file or directory
debug2: we did not send a packet, disable method
debug3: authmethod_lookup password
debug3: remaining preferred: ,password
debug3: authmethod_is_enabled password
debug1: Next authentication method: password
```

See below:
<img width="1479" alt="image" src="https://github.com/user-attachments/assets/c4e20047-a56a-446e-acf5-61e34dd7ab07">

Any guidance will be highly appreciated.

**Comment by diraneyya** [comment]
TLDR: pico.sh is accepting my key but tuns.sh is not

`ssh -n -v pico.sh`:
```
debug1: Offering public key: /root/.ssh/id_rsa RSA SHA256:Nn432Ci4+oukTQK7nNeTtjzaKM3JLjL9b50kXqeeI+0
debug1: Server accepts key: /root/.ssh/id_rsa RSA SHA256:Nn432Ci4+oukTQK7nNeTtjzaKM3JLjL9b50kXqeeI+0
Authenticated to pico.sh ([141.148.85.132]:22) using "publickey".
```

`ssh -n -v tuns.sh`:
```
debug1: Offering public key: /root/.ssh/id_rsa RSA SHA256:Nn432Ci4+oukTQK7nNeTtjzaKM3JLjL9b50kXqeeI+0
debug1: Authentications that can continue: password,publickey
debug1: Trying private key: ...
debug1: Next authentication method: password
```

Can you please help.

**Comment by diraneyya** [comment]
Hello @neurosnap thank you for the reply. 

The management application `ssh pico.sh` which allows you to add your certificates made it seem as if the analytics and a few other features needed a subscription, rather than the core service.

My advice, make that clear in the interface, that any Pico service is a Pico+ service, disallow adding certificates without a subscription, or indicate when connecting with an added certificate not without a subscription that a subscription will be needed.

It's misleading. 

Orwa.

**Comment by diraneyya** [comment]
Hello @neurosnap thank you for the reply. 

The management application `ssh pico.sh` which allows you to add your certificates made it seem as if the analytics and a few other features needed a subscription, rather than the core service.

My advice, make that clear in the interface, that any Pico service is a Pico+ service, disallow adding certificates without a subscription, or indicate when connecting with an added certificate not without a subscription that a subscription will be needed.

It's misleading. 

Orwa.

**Comment by diraneyya** [comment]
In all honesty. I have nearly paid for a subscription because I appreciated the product and thought that two dollars per month was nothing. The only reason I held off is because I wanted to measure the tunnel speeds before and after the subscription. 

It's certainly not properly communicated.

**Comment by neurosnap** [comment]
Thank you for your feedback!  We are always trying to make it more clear what are free versus paid services but clearly need to make improvements. I'll add your comments to my shortlist to prioritize.

**Comment by antoniomika** [comment]
Hey @diraneyya ,

Wanted to check where was the confusion coming from? At least any ingress points to tuns from the sish docs mention it's a managed service and has a note box explaining it's a premium pico+ service. I think we can do a better job outputting an error message when connecting to tuns without a plus account for sure but want to see how we can make the copy better.

Gonna close this but we can still discuss on this issue.

**Comment by diraneyya** [comment]
I am assuming that you are referring to this blue box:
<img width="695" alt="image" src="https://github.com/user-attachments/assets/1ea60fd2-0bd5-4666-90b1-c85a0e9569a2" />

The blue box enough does not convey the reality of the situation.

There are a dozen ways you can make this clearer:
1. In the `ssh pico.sh` interactive menu (which is quite neat), consider these modifications for additional clariy:
  - remove the Pico+ option, and completely rework the Pico+ branding, if the only kind of access is Pico+ access, then it is not a premium and no '+' is needed. This is just basic access.
  - Remove any indication of "full access", it implies there is a "basic access"
  - Instead of *Username* and *Joined* indicate that the user is *Unactive* until they have a subscription
  - When attempting to add certificates, highlight that they won't be able to use the basic service until they have a subscriptoion
1. In the `ssh ... tuns.sh` call, differentiate between users attempting to connect after having done all the right steps, acknowledging that this is the point in time where they would have otherwise enjoy the service, have they subscribed
1. Consider allowing the new users to connect a few times (let's say 10) before you tell them that their trial phase ended. Not only that this will not cost you much, it would have (in my case) made me pay for a subscription instead of being ticked off

Additionally, remember that you are competing with things like this [telebit.io](https://telebit.cloud/) in the public domain and cloudflared tunnels in the private domain, BOTH of which are free of charge.

In fact, I would have aid for [telebit.io](https://telebit.cloud/) without hesitation if the person behind it provided a paid tier. Simply because I love his work and read his shell scripts as if they are poetry.

If you need more feedback or have questions, please do not hesitate to ask.

**Comment by neurosnap** [comment]
Thank you for the feedback, this is great.  I'll incorporate some of your changes because I do think our docs and messaging can be improved.  Much appreciated
